janusgraph-core/src/main/java/org/janusgraph/graphdb/tinkerpop/JanusGraphFeatures.java

```diff
@@ -78,7 +78,7 @@ public static JanusGraphFeatures getFeatures(StandardJanusGraph graph, StoreFeat
 
         @Override
         public boolean supportsMapValues() {
-            return false;
+            return true;
```

Comment:
This update was to resolve test failures and involved adding an initial implementation for HashMap serialization. The implementation simply uses Java object serialization, which will throw a `java.io.NotSerializableException` if the map contains unsupported types.

As noted in the PR notes there are definitely at least performance issues with using Java serialization here. Can we create a separate issue to update the initial HashMap serialization implementation to both improve performance and formalize validation?